crates/uv-settings/src/lib.rs

```diff
@@ -185,6 +199,48 @@ fn config_dir() -> Option<PathBuf> {
     }
 }
 
+fn locate_system_config_xdg() -> Option<PathBuf> {
+    // On Linux/MacOS systems, read the XDG_CONFIG_DIRS environment variable
+    if let Ok(config_dirs) = std::env::var("XDG_CONFIG_DIRS") {
```

Comment:
I went to re-read the spec, which says

> If $XDG_CONFIG_DIRS is either not set or empty, a value equal to /etc/xdg should be used.

I think this means don't fall back to /etc/xdg if the env var contains even just `:`

If that's true you should handle the "/etc/xdg" default here and not in `system_config_file()`.  To make things easier you could pass in the env var (so you don't have to set it in the test), since this is only used internally.